source/faq.txt

```diff
@@ -148,6 +148,26 @@ When ``MongoClient.close()`` is called by any thread, the driver
 closes all idle sockets and closes all sockets that are in
 use as they are returned to the pool.
 
+Why does the application throw "IllegalStateException: state should be: open"?
```

Comment:
Suggestion: make the question more oriented around the user

```suggestion
Why did I receive an "IllegalStateException: state should be: open" error?
```

Reply:
Thanks, I reworded since "receive" doesn't seem like a common action to describe exceptions.